src/style.css

```diff
@@ -0,0 +1,140 @@
+@import "./stars.css";
+
+.productCard__card {
+  width: 198px;
+  height: 406px;
+
+}
+
+.card {
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.image {
+  background-image: url("./images/imac.jpeg");
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+}
+
+.card__title {
+  margin: 40px 16px 0;
+}
+
+.title {
+  font-size: 12px;
+  line-height: 18px;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+  font-weight: 500;
+}
+
+.card__code {
+  margin: 4px 16px 0;
+}
+
+.code {
+  font-size: 10px;
+  line-height: 14px;
+  font-family: "Roboto", sans-serif;
+  color: #616070;
+  font-weight: 400;
+}
+
+.stars {
+  display: flex;
+  margin: 0 17px 0 16px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

Comment:
You have separate file for stars then why you put this here??. Please revisit

Reply:
you're right. 
I deleted this irrelevent codes from my style.css and arranged everything in such a way that it led me get rid of any styling about stars.. 
now I am only importing it and everything works perfectly. 

afterwards I don't mix tasks with eachother